src/Input/Input.tsx

```diff
@@ -22,16 +22,16 @@ export interface InputProps {
  * in this component and should be handled in the consuming app.
  *
  */
-const Input: FC<InputProps & ComponentPropsWithoutRef<'input'>> = ({ label, error = '', ...props }) => {
-    return (
+const Input = forwardRef<HTMLInputElement, InputProps & ComponentPropsWithoutRef<'input'>>(
```

Comment:
I don't think the HTMLInputElement is required if you keep the `FC` generics and use  `ComponentPropsWithRef` [(see TextArea component with ref)](https://github.com/ToyotaResearchInstitute/lakefront/blob/main/src/TextArea/TextArea.tsx).

Reply:
Done. Thanks for the suggestion.